addons/binding/org.openhab.binding.enigma2/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,27 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: Enigma2 Binding
+Bundle-SymbolicName: org.openhab.binding.enigma2;singleton:=true
+Bundle-Vendor: openHAB
+Bundle-Version: 2.1.0.qualifier
+Bundle-RequiredExecutionEnvironment: JavaSE-1.7
+Bundle-ClassPath: .
+Import-Package: 
+ javax.jmdns;version="3.5.1",

```

Comment:
do you need this specific version of jmdns? If not, please remove version constraint.